indico/modules/events/registration/templates/display/_registration_summary_blocks.html

```diff
@@ -99,6 +99,8 @@
         </ul>
     {% elif field.input_type == 'file' and friendly_data and from_management %}
         <a href="{{ url_for('.registration_file', data[field.id].locator.file) }}">{{ friendly_data }}</a>
+    {% elif field.input_type == 'picture' and friendly_data %}
+        <img class="picture-preview" src="{{ url_for('.registration_picture', data[field.id].locator.file) }}" alt={{ friendly_data }}>
```

Comment:
does the `alt` have anything useful in it? Also, it almost certainly needs to be quoted

```suggestion
        <img class="picture-preview" src="{{ url_for('.registration_picture', data[field.id].locator.file) }}" alt="{{ friendly_data }}">
```

Reply:
The information in the alt is the uploaded file name.